src/main/scala/io/iohk/ethereum/crypto/package.scala

```diff
@@ -45,6 +46,24 @@ package object crypto {
     generator.generateKeyPair()
   }
 
+  /** @return (privateKey, publicKey) pair */
+  def keyPairToByteArrays(keyPair: AsymmetricCipherKeyPair): (Array[Byte], Array[Byte]) = {
```

Comment:
I agree, it seems better to remove `AsymmetricCipherKeyPairSerializable` and save all the keys in the same format with only the private key :+1: